src/Symfony/Component/Form/Extension/Validator/Type/FormTypeValidatorExtension.php

```diff
@@ -58,9 +60,18 @@ public function configureOptions(OptionsResolver $resolver)
             'constraints' => [],
             'invalid_message' => 'This value is not valid.',
             'invalid_message_parameters' => [],
+            'legacy_error_messages' => $this->legacyErrorMessages,
             'allow_extra_fields' => false,
             'extra_fields_message' => 'This form should not contain extra fields.',
         ]);
+        $resolver->setAllowedTypes('legacy_error_messages', 'bool');
+        $resolver->setDeprecated('legacy_error_messages', function (OptionsResolver $resolver, $value) {
+            if ($value === true) {
+                return 'Setting the option \'legacy_error_messages\' to \'true\' is deprecated and will be disabled by default in Symfony 5.0';
```

Comment:
I would use double quotes here too for enclosing option names and values (same as above)